test/unit/client/test_client.py

```diff
@@ -145,6 +151,19 @@ def search_response(file_id, folder_id):
     return mock_network_response
 
 
+@pytest.fixture(scope='module')
+def recent_items_response(file_id, marker_id):
+    mock_network_response = Mock(DefaultNetworkResponse)
+    mock_network_response.json.return_value = {
+        'entries': [
+            {'type': 'recent_item', 'item': {'type': 'file', 'id': file_id}}
+        ],
+        'next_marker': marker_id,
```

Comment:
Providing a `next_marker` implies that there are more results coming. So if you try paging, it'll try to make a second API call, which doesn't seem like what you want.